sql-cli/include/base/config/dev/configuration.yml

```diff
@@ -3,7 +3,7 @@ connections:
 ## Example of SQLite connection
   - conn_id: sqlite_conn
     conn_type: sqlite
-    host: $PWD/data/retail.db
+    host: $PWD/data/dev_movies.db
```

Comment:
:+1: I'm happy to drop the retail example for now - we can always use it in future if we need it